camayoc/tests/qpc/api/v1/utils.py

```diff
@@ -12,6 +13,108 @@
     FailedScanException,
     WaitTimeError,
 )
+from camayoc.qpc_models import (
+    Credential,
+    Scan,
+    Source
+)
+
+
+def get_source(source_type, cleanup):
+    """Retrieve a single network source if available from config file.
+
+    Retreive one network source from the config file.
```

Comment:
I think this line can be dropped, it pretty much says the same as the line above